openedx_events/content_authoring/signals.py

```diff
@@ -210,3 +211,36 @@
         "content_object": ContentObjectData
     }
 )
+
+# .. event_type: org.openedx.content_authoring.content.library.collection.created.v1
+# .. event_name: LIBRARY_COLLECTION_CREATED
+# .. event_description: emitted when a content library collection is created
+# .. event_data: LibraryCollectionData
+LIBRARY_COLLECTION_CREATED = OpenEdxPublicSignal(
```

Comment:
@bradenmacdonald Are we allowed to put new signals in this repo? Or are we supposed to put them in https://github.com/openedx/openedx-events ?

Reply:
I'm actually not sure. Maybe @mariajgrimaldi can answer? What's the difference between this `signals.py` and that [`signals.py`](https://github.com/openedx/openedx-events/blob/main/openedx_events/content_authoring/signals.py)?

Reply:
I'm not sure I follow. This repo es `openedx-events` and both signals.py files are the same.

You are definitely allowed to propose new signal definitions in this repository and the location is correct. The idea of having the signal here instead of in the edx-platform code is that plugins that will listen for the signal can include this lightweight library in their dependencies instead of requiring edx-platform all the time.

Reply:
lol whoops, please ignore my confusion.

Reply:
No worries. Happy to address this in depth. As before, if you are planning to add a signal into edx-platform, then it would be better to write it here so plugins can use that without the large import.
However, other pattern we have seen is when a plugin defines a new signal and uses it in its own code. In that case if reuse of said signal is not a goal from the devs, then the definition can be left in the plugin as you could import that plugin as a library if you are writing a second level library/plugin.

Reply:
My apologies for the noise! I could have sworn there was a signals.py file in openedx-learning, but it's not there now, so not sure what I was on about :)

Reply:
Ah, it was here! [signals.py](https://github.com/openedx/openedx-learning/pull/131/files#diff-d3efcf03e0d50e9d250f3fe390cdf827b7373f7e4db4466d07c3cc46825296c1). I'm not going nuts.. 😊